### ISSUE TITLE
Property changed value but changed fields is only "bytes"

### ISSUE THREAD

**Issue by oatspear** [gamehook-io/gamehook#31]
**Expected:** When a property changes its value, the signalR message should include `values` among its `changed_fields`.

**Observed:** `changed_fields` contains only `bytes` in some cases.

**Example:**

1. Load Pokémon Yellow within RetroArch. Ensure that there is a previous save file. Pause the game before the main menu shows up.
2. Load GameHook's latest version with the official Pokémon Yellow mapper.
3. Start external software, connecting to GameHook via signalR and handling property change events.
4. With everything running, resume the game and enter the main menu (Continue, New Game, Options).

When entering the main menu, the game loads the previous save file and a bunch of properties, such as Pokémon species and moves, are automatically loaded as well. GameHook's web interface reflects this, and shows their values correctly.

Example callback function (Python):

```python
def _on_property_changed(self, args):
    prop, _address, value, byte_values, _frozen, changed_fields = args
    if prop.endswith('species') or prop.endswith('move1'):
        logger.warning(f'GameHook changed {prop}: {value} {byte_values}')
        logger.warning(f'  > changed fields: {changed_fields}')
    if 'value' in changed_fields:
        self.on_change(prop, value, byte_values)
```

Observed output when the main menu loads:

> [WARNING] gamehook (2022-08-17 08:38:15,381): GameHook changed player.team.0.species: {'pokedexNumber': '25', 'name': 'Pikachu'} [84]
> [WARNING] gamehook (2022-08-17 08:38:15,382):   > changed fields: ['bytes']
> [WARNING] gamehook (2022-08-17 08:38:15,383): GameHook changed player.team.0.move1: THUNDERSHOCK [84]
> [WARNING] gamehook (2022-08-17 08:38:15,384):   > changed fields: ['bytes']

As you can see, the properties changed both value and bytes, but only `bytes` is triggered.

**Comment by flamesage** [comment]
This should be fixed in 2022.8.23.1. Please confirm.

**Comment by flamesage** [comment]
Closed due to inactivity. Confirmed this was fixed. Please reopen if you have any additional questions.
